packages/console/src/pages/SignInExperience/components/ManageLanguageModal/index.tsx

```diff
@@ -26,31 +26,30 @@ const ManageLanguageModal = ({ isOpen, onClose }: ManageLanguageModalProps) => {
     '/api/custom-phrases'
   );
 
-  const allLanguageKeys = useMemo(() => {
-    const uiBuiltInLanguageKeys = uiLanguageOptions.map((option) => option.value);
-    const customUiLanguageKeys = customPhraseResponses?.map(({ languageKey }) => languageKey);
+  const allLanguageTags = useMemo(() => {
+    const customUiLanguageTags = customPhraseResponses?.map(({ languageKey }) => languageKey);
 
-    const allKeys = customUiLanguageKeys?.length
-      ? [...new Set([...uiBuiltInLanguageKeys, ...customUiLanguageKeys])]
-      : uiBuiltInLanguageKeys;
+    const allTags = customUiLanguageTags?.length
+      ? [...new Set([...builtInUiLanguages, ...customUiLanguageTags])]
+      : builtInUiLanguages;
 
-    return allKeys.slice().sort();
+    return allTags.slice().sort();
   }, [customPhraseResponses]);
 
-  const defaultLanguageKey = getDefaultLanguage(allLanguageKeys[0] ?? '');
+  const defaultLanguageTag = getDefaultLanguageTag(allLanguageTags[0] ?? '');
```

Comment:
assume that the first custom phrase language key is zh-TW (`allLanguageTags[0] === 'zh-TW'`),
and `getDefaultLanguageTag(allLanguageTags[0] ?? '')` will be `en`.
Is it expected?

Reply:
updated.